packages/headless/doc-parser/src/interface-resolver.test.ts

```diff
@@ -487,12 +488,60 @@ describe('#resolveInterfaceMembers', () => {
     entry.addMember(pager);
     entry.addMember(controller);
 
-    const result = resolveInterfaceMembers(entry, pager);
+    const result = resolveInterfaceMembers(entry, pager, []);
     const expected = buildMockEntity({
       name: 'state',
       type: 'string',
     });
 
     expect(result).toEqual([expected]);
   });
+
+  it(`an interface has a property of its own type,
+  the interface's members are parsed only twice`, () => {
```

Comment:
Super nice unit test. Could we also add a comment explaining by twice?

Reply:
Do you mean a more descriptive name for the test? I'll do that

Reply:
I think the test description is really good, I was just thinking a javascript comment with `//` giving some additional context.